libff/algebra/curves/public_params.hpp

```diff
@@ -43,22 +44,19 @@ namespace libff {
   affine_ate_G2_precomp<EC_ppT> affine_ate_precompute_G2(const G2<EC_ppT> &Q);
 
 
-  Fqk<EC_ppT> affine_ate_miller_loop(const affine_ate_G1_precomp<EC_ppT> &prec_P,
-                                     const affine_ate_G2_precomp<EC_ppT> &prec_Q);
-  Fqk<EC_ppT> affine_ate_e_over_e_miller_loop(const affine_ate_G1_precomp<EC_ppT> &prec_P1,
-                                              const affine_ate_G2_precomp<EC_ppT> &prec_Q1,
-                                              const affine_ate_G1_precomp<EC_ppT> &prec_P2,
-                                              const affine_ate_G2_precomp<EC_ppT> &prec_Q2);
-  Fqk<EC_ppT> affine_ate_e_times_e_over_e_miller_loop(const affine_ate_G1_precomp<EC_ppT> &prec_P1,
-                                                      const affine_ate_G2_precomp<EC_ppT> &prec_Q1,
-                                                      const affine_ate_G1_precomp<EC_ppT> &prec_P2,
-                                                      const affine_ate_G2_precomp<EC_ppT> &prec_Q2,
-                                                      const affine_ate_G1_precomp<EC_ppT> &prec_P3,
-                                                      const affine_ate_G2_precomp<EC_ppT> &prec_Q3);
-  Fqk<EC_ppT> double_miller_loop(const G1_precomp<EC_ppT> &prec_P1,
-                                 const G2_precomp<EC_ppT> &prec_Q1,
-                                 const G1_precomp<EC_ppT> &prec_P2,
-                                 const G2_precomp<EC_ppT> &prec_Q2);
+  Fqk<EC_ppT> affine_ate_miller_loop(const affine_ate_G1_precomp<EC_ppT>
```

Comment:
Let's try to enforce the format of this comment, else it is very hard to read.

Reply:
Used the clang-formatted version of the original.